cgi-bin/LJ/Faq.pm

```diff
@@ -493,34 +493,22 @@ sub load_matching {
         or die "initial FAQ rendering failed";
 
     foreach my $f (@faqs) {
-	my $score = 0;
-
-	if ($f->question_raw =~ /\Q$term\E/i) {
-	    $score += 3;
-	}
-	if ($f->question_raw =~ /\b\Q$term\E\b/i) {
-	    $score += 5;
-	}
-
-	if ($f->summary_raw =~ /\Q$term\E/i) {
-	    $score += 2;
-	}
-	if ($f->summary_raw =~ /\b\Q$term\E\b/i) {
-	    $score += 4;
-	}
-
-	if ($f->answer_raw =~ /\Q$term\E/i) {
-	    $score += 1;
-	}
-	if ($f->answer_raw =~ /\b\Q$term\E\b/i) {
-	    $score += 3;
-	}
-
-	next unless $score;
-
-	$scores{$f->{faqid}} = $score;
-
-	push @results, $f;
+        my $score = 0;
+
+        $score += 3 if $f->question_raw =~ /\Q$term\E/i;
```

Comment:
I love when we find things that horrify you, Mark.

IIRC, worst username evar hacked this together for support in a weekend because I was getting increasingly desperate!